docs/changelog-fragments.d/805.deprecation.rst

```diff
@@ -0,0 +1,12 @@
+Deprecated :py:meth:`~cheroot.makefile.MakeFile` in favor of :py:class:`~cheroot.makefile.StreamReader`
```

Comment:
Pretty sure it's a function:
```suggestion
Deprecated :py:func:`~cheroot.makefile.MakeFile` in favor of :py:class:`~cheroot.makefile.StreamReader`
```
*(although, Sphinx doesn't seem to mind: https://cheroot--805.org.readthedocs.build/en/805/history/#deprecations-removal-in-next-major-release)*